code/clue.js

```diff
@@ -3,43 +3,273 @@
+  id: 'rope'
+}
+const knife = {
+  name: 'Knife',
+  weight: 2,
+  color: '#a19c85',
+  mortality: 6,
+  id: 'knife'
+}
+const candlestick = {
+  name: 'Candlestick',
+  weight: 30,
+  color: '#737275',
+  mortality: 4,
+  id: 'candlestick'
+}
+const dumbbell = {
+  name: 'Dumbbell',
+  weight: 90,
+  color: '#aaa7b0',
+  mortality: 5,
+  id: 'dumbbell'
+}
+const poison = {
+  name: 'Poison',
+  weight: 3,
+  color: '#6e9692',
+  mortality: 8,
+  id: 'poison'
+}
+const axe = {
+  name: 'Axe',
+  weight: 45,
+  color: '#7d9e98',
+  mortality: 7,
+  id: 'axe'
+}
+const bat = {
+  name: 'Bat',
+  weight: 30,
+  color: '#909c76',
+  mortality: 5,
+  id: 'bat'
+}
+const trophy = {
+  name: 'Trophy',
+  weight: 50,
+  color: '#a8ad7d',
+  mortality: 3,
+  id: 'trophy'
+}
+const pistol = {
+  name: 'Pistol',
+  weight: 70,
+  color: '#949081',
+  mortality: 10,
+  id: 'pistol'
 }
 
 // THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
 
+const diningRoom = {
+  name: 'Dining Room',
+  color: '#737260'
+
+}
+const conservatory = {
+  name: 'Conservatory',
+  color: '#6e8264'
+}
+
+const kitchen = {
+  name: 'Kitchen',
+  color: '#77637d'
+}
+
+const study = {
+  name: 'Study',
+  color: '#d1bda9'
+}
+
+const library = {
+  name: 'Library',
+  color: '#75685b'
+}
+
+const billiardRoom = {
+  name: 'Billiard Room',
+  color: '#804f62'
+}
+
+const lounge = {
+  name: 'Lounge',
+  color: '#8a895c'
+}
+
+const ballRoom = {
+  name: 'Ballroom',
+  color: '#697c91'
+}
+
+const hall = {
+  name: 'Hall',
+  color: '#99708b'
+}
+
+const spa = {
+  name: 'Spa',
+  color: '#82745f'
+}
+
+const livingRoom = {
+  name: 'Living Room',
+  color: '#c7ccb4'
+}
+
+const observatory = {
+  name: 'Observatory',
+  color: '#e3d4bc'
+}
+
+const theater = {
+  name: 'Theater',
+  color: '#608281'
+}
+
+const guestHouse = {
+  name: 'Guest House',
+  color: '#655c80'
+}
+
+const patio = {
+  name: 'Patio',
+  color: '#755b70'
+}
+
+
+
 // NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  mrsWhite,
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard
+
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
 
-const rooms = []
+const rooms = [
+  diningRoom,
+  conservatory,
+  kitchen,
+  study,
+  library,
+  billiardRoom,
+  lounge,
+  ballRoom,
+  hall,
+  spa,
+  livingRoom,
+  observatory,
+  theater,
+  guestHouse,
+  patio
+]
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
 // YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
+
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
 // CREATE AN OBJECT THAT KEEPS THE MYSTERY.
 // With a killer, a weapon and a room.
 // The values will be set later.
+let mystery = {
+  killer: '',
+  weapon: '',
+  room: ''
```

Comment:
Super small thing, but I would suggest you to initialise your mystery object with null values rather than empty strings. It's about sending proper information for other developers : null value indicates that probably we will replace these values with something else, but without saying what type will that new value have for now. Initialising your object with empty string values ' ' could be misunderstood by some one in a way that your properties will later on by replaced by new strings only. As a sum up, this would be better :
```
const mystery = {
  killer: null,
  weapon: null,
  room: null,
}
```